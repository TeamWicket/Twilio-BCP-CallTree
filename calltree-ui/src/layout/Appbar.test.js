import CustomAppBar, { CustomUserMenu, ConfigurationMenu } from './AppBar';
import React from 'react';
import { shallow } from 'enzyme';

describe("App bar", () => {
    it("should match snapshot of Custom AppBar", () => {
        expect(shallow(<CustomAppBar />)).toMatchSnapshot();
    })
    it("should match snapshot of Custom User Menu", () => {
        expect(shallow(<CustomUserMenu />)).toMatchSnapshot();
    })
    it("should match snapshot of ConfigurationMenu", () => {
        expect(shallow(<ConfigurationMenu />)).toMatchSnapshot();
    })
})
