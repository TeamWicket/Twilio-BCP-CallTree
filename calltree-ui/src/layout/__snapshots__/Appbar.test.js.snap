// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App bar should match snapshot of ConfigurationMenu 1`] = `ShallowWrapper {}`;

exports[`App bar should match snapshot of Custom AppBar 1`] = `ShallowWrapper {}`;

exports[`App bar should match snapshot of Custom User Menu 1`] = `ShallowWrapper {}`;
